regression/verilog/SVA/cycle_delay1.desc

```diff
@@ -0,0 +1,10 @@
+CORE
+cycle_delay1.sv
+
+^C string exception : failed to convert sva_cycle_delay offsets$
+^EXIT=6$
+^SIGNAL=0$
+--
+^warning: ignoring
+--
+The error message could be better.
```

Comment:
While this is true I am worried no one will ever pay attention to it again? Perhaps now is the time to fix it?

Reply:
Done